### PR TITLE
BenMens/issue27

### PR DESCRIPTION
**Pull request by BenMens**

### CODE REVIEW

File: CellEditorView.pde [hunk 1]
```diff
@@ -1,35 +1,92 @@
...
 
-        this.boundsRect = new Rectangle2D.Float(0, 0, 200, 300);
         this.shouldClip = true;
-    }
 
-    void registerClient(CellEditorViewClient client) {
-        if(!clients.contains(client)) {
-            clients.add(client);
-        }  
+        font = createFont("courrier.dfont", 24);
```
Comment (basmens) [inline CellEditorView.pde:16]: can you move this to the to setup function

File: ViewBase.pde [hunk 2]
```diff
@@ -380,6 +414,37 @@ class ViewBase {
         return false;
     }
 
+    // ########################################################################
+    // FrameRect
```
Comment (basmens) [inline ViewBase.pde:418]: can you change this to 'set frame rect'

File: ViewBase.pde [hunk 3]
```diff
@@ -380,6 +414,37 @@ class ViewBase {
...
+
+    Rectangle2D.Float getFrameRect() {
+        return (Rectangle2D.Float)frameRect.clone();
+    }
+
+    void onFrameRectChange(Rectangle2D.Float oldRect) {}
+
+
+    // ########################################################################
+    // Bounds
```
Comment (basmens) [inline ViewBase.pde]: can you change this to 'set bounds rect'

File: ViewBase.pde (hunk 2 above)
Comment (BenMens) [reply ViewBase.pde:418]: No

File: CellEditorView.pde (hunk 1 above)
Comment (BenMens) [reply CellEditorView.pde:16]: Already done

**Review by basmens — APPROVED** [review]
well done!

**Pull request merged**